build.gradle

```diff
@@ -154,7 +154,7 @@ dependencies {
 
   implementation 'com.github.hmcts.ccd-case-migration-starter:domain:3.0.0'
   implementation 'uk.gov.hmcts.reform.ccd-case-migration:processor:DEV-SNAPSHOT'
-  implementation 'com.github.hmcts:sscs-common:4.9.3'
+  implementation 'com.github.hmcts:sscs-common:4.9.4-RC-SSCS-10517-2'
```

Comment:
your commons version looks a bit out of date?

Reply:
It is very out of date.. I'm waiting for all this MVP shizz to go through then I'll update it.